src/application.coffee

```diff
@@ -270,7 +274,9 @@ application.lockUpdates = lockUpdates = do ->
 				.catch ENOENT, _.noop
 				.catch (err) ->
 					release()
-					throw new Error("Updates are locked: #{err.message}")
+					err = new Error("Updates are locked: #{err.message}")
+					err.isLocked = true
```

Comment:
It would be nicer to use a `typed-error` here and `throw new UpdatesLockedError("Updates are locked: #{err.message}")` or similar